dept44-starter/src/main/java/se/sundsvall/dept44/logbook/filter/json/JsonPathBodyFilters.java

```diff
@@ -0,0 +1,197 @@
+package se.sundsvall.dept44.logbook.filter.json;
+
+import static com.jayway.jsonpath.JsonPath.compile;
+import static org.apiguardian.api.API.Status.EXPERIMENTAL;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.function.Predicate;
+import java.util.function.UnaryOperator;
+import java.util.regex.Pattern;
+
+import javax.annotation.Nullable;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.BooleanNode;
+import com.fasterxml.jackson.databind.node.DoubleNode;
+import com.fasterxml.jackson.databind.node.NullNode;
+import com.fasterxml.jackson.databind.node.TextNode;
+import com.jayway.jsonpath.Configuration;
+import com.jayway.jsonpath.DocumentContext;
+import com.jayway.jsonpath.JsonPath;
+import com.jayway.jsonpath.Option;
+import com.jayway.jsonpath.ParseContext;
+import com.jayway.jsonpath.spi.json.JacksonJsonProvider;
+import com.jayway.jsonpath.spi.mapper.JacksonMappingProvider;
+import org.apiguardian.api.API;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.zalando.logbook.BodyFilter;
+
+/**
+ * Workaround copy of the original {@link org.zalando.logbook.json.JsonPathBodyFilters} and
+ * {@link org.zalando.logbook.json.JsonMediaType#JSON} that uses {@code JacksonJsonProvider} instead
+ * of {@code JacksonJsonNodeJsonProvider} in order to avoid NPE:s due to a bug in JsonPath (see
+ * https://github.com/zalando/logbook/issues/1369 and https://github.com/json-path/JsonPath/issues/678).
+ *
+ * Should be removed when the above is fixed.
+ */
```

Comment:
Maybe put a `TODO ` here as well? 
Otherwise we will probably forget to rmeove this in the future?